tests/test_preview.py

```diff
@@ -162,9 +162,9 @@ def test_preview_to_file(test_dir, wlmutils):
     assert path.is_file()
```

Comment:
I think we need to add a `pytest.mark.group_xxx` to this file to ensure it isn't skipped.

Based on that, I'd actually like to see this file split in two:
1. tests that require a WLM/allocation (perhaps `tests/on_wlm/test_preview_active_infra.py`)
2. tests that can complete without (current file)

- Items in (1) can be added to `pytestmark = pytest.mark.slow_tests`
- Items in (2) can be added to `pytestmark = pytest.mark.group_b`

Known "active" tests that fail for me are:
- FAILED tests/test_preview.py::test_active_orchestrator_jobs_property
- FAILED tests/test_preview.py::test_preview_active_infrastructure
- FAILED tests/test_preview.py::test_preview_orch_active_infrastructure
- FAILED tests/test_preview.py::test_preview_multidb_active_infrastructure
- FAILED tests/test_preview.py::test_preview_active_infrastructure_orchestrator_error

Reply:
note: there shouldn't be any reason to require an allocation to run `test_get_ifname_filter` - let's modify that test accordingly.

An option might be:

```py
value = ["_ifname=bad_if_key", "ifname=mock_if_name", "IFname=case_sensitive_key"]
template = "{{value | get_ifname}}"
FILTERS["get_ifname"] = previewrenderer.get_ifname

output = Template(template_str).render(value=value)

assert "mock_if_name" in output
```

We should also add tests establishing expectations for the other test cases shown above in `value`:
- line not starting with ifname
- IFname comparison case sensitivity).

Reply:
resolved on a call.